src/report_workflow/test_report_runner.py

```diff
@@ -236,7 +236,7 @@ def get_failed_tests(base_path: str, test_number: str, test_type: str, component
         return failed_test
 
     if result_content:
-        if ("infoBox success" in result_content) and ("<h2>Failed tests</h2>" not in result_content):
+        if ("<h2>Failed tests</h2>" not in result_content):
```

Comment:
@peterzhuamazon can we use `failed_tests` in the yaml?

Reply:
Not getting this comment, failed_test will only show once we parse the html, and this is using the html to know the status of what to add to failed_test.